### PR TITLE
feat(edpa): throttle VID recovery RPCs

### PR DESCRIPTION
**Pull request by Marco-Premier**
Throttle dead-letter and manual recovery RPCs and document the deployment-wide rate budget.

Issue: #4427

### CODE REVIEW

**Comment by wfa-reviewable** [comment]
This change is [Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4429)

File: src/main/kotlin/org/wfanet/measurement/edpaggregator/tools/VidLabelingHeal.kt [hunk 1]
```diff
@@ -189,6 +190,27 @@ class MarkFailedCommand : EdpaApiCommand() {
   mixinStandardHelpOptions = true,
 )
 class RetryFailedCommand : EdpaApiCommand() {
+  @Option(
+    names = ["--metadata-read-rpc-min-interval"],
```
Comment (stevenwarejones) [inline src/main/kotlin/org/wfanet/measurement/edpaggregator/tools/VidLabelingHeal.kt:195]: Use the strict VID duration converter added in #4440 for all three RPC interval options. The current Duration conversion still uses substring parsing: I verified that both `-1s` and `500msjunk` pass CLI parsing and proceed to required-option validation, becoming positive durations instead of being rejected.

Extract `VidLabelingRpcDurationConverter` into a small reusable Bazel target, apply it to these options, and add command-line parsing tests covering both values.

File: src/main/kotlin/org/wfanet/measurement/securecomputation/deploy/gcloud/spanner/InternalApiServer.kt [hunk 2]
```diff
@@ -166,6 +167,27 @@ class InternalApiServer : Runnable {
   )
   private lateinit var channelShutdownTimeout: Duration
 
+  @CommandLine.Option(
+    names = ["--metadata-read-rpc-min-interval"],
```
Comment (stevenwarejones) [inline src/main/kotlin/org/wfanet/measurement/securecomputation/deploy/gcloud/spanner/InternalApiServer.kt:172]: Apply the strict VID duration converter to these three options as well. I verified that `--metadata-read-rpc-min-interval=-1s` passes this server's CLI parsing, so malformed configuration can silently become a positive interval.

Expose the reusable converter target to the Secure Computation deployment package and add command-line tests for negative and partially malformed values.

**Review by stevenwarejones — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4429)* status: 0 of 13 files reviewed, 2 unresolved discussions (waiting on Marco-Premier and stevenwarejones).

File: docs/edpaggregator/deployment-guide.md [hunk 3]
```diff
@@ -656,6 +656,54 @@ supply:
 Leave `vid_labeling_workers = {}` to skip the phase workers/queues; provide worker
 entries only once your market has adopted VID labeling.
 
+#### VID Labeling outbound RPC rate limits
+
+Every VID Labeling process shares four client-side throttlers across its coroutines. The defaults
```
Comment (stevenwarejones) [inline docs/edpaggregator/deployment-guide.md:661]: Update this rate budget to match the current defaults. The configured intervals are 500ms, 100ms, 200ms, and 250ms, which correspond to 2 QPS for Kingdom, 10 QPS for metadata reads, 5 QPS for metadata writes, and 4 QPS for the control plane—not 2/2/1/2 QPS.

The 84 metadata RPC/s and 56 control-plane RPC/s calculations below are consequently stale. Recalculate the deployment envelope from the actual defaults and process topology, and do not claim it remains below a per-method 100-QPS limit unless that bound is demonstrated for each affected method.

**Review by stevenwarejones — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4429)* status: 0 of 13 files reviewed, 3 unresolved discussions (waiting on Marco-Premier and stevenwarejones).

File: docs/edpaggregator/deployment-guide.md [hunk 4]
```diff
@@ -656,6 +656,54 @@ supply:
...
+per_method_rate_limit {
+  key: "wfa.measurement.api.v2alpha.ModelLines/ListModelLines"
+  value {
+    maximum_request_count: 20
+    average_request_rate: 5
+  }
+}
+```
+
+Both VID Labeling scheduler jobs use a 600-second attempt deadline matching the monitor timeout. The
```
Comment (stevenwarejones) [inline docs/edpaggregator/deployment-guide.md]: Correct this to 660 seconds. Terraform sets `attempt_deadline = "660s"` for both VID Labeling scheduler jobs so Scheduler has 60 seconds of response headroom beyond the monitor function's 600-second timeout. Saying the deadline is 600 seconds and matches the timeout contradicts the deployed configuration and removes the rationale for the extra headroom.

**Review by stevenwarejones — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4429)* status: 0 of 13 files reviewed, 4 unresolved discussions (waiting on Marco-Premier and stevenwarejones).

File: docs/edpaggregator/deployment-guide.md (hunk 3 above)
Comment (Marco-Premier) [reply docs/edpaggregator/deployment-guide.md:661]: Done.

File: docs/edpaggregator/deployment-guide.md (hunk 4 above)
Comment (Marco-Premier) [reply docs/edpaggregator/deployment-guide.md]: Done.

File: src/main/kotlin/org/wfanet/measurement/edpaggregator/tools/VidLabelingHeal.kt (hunk 1 above)
Comment (Marco-Premier) [reply src/main/kotlin/org/wfanet/measurement/edpaggregator/tools/VidLabelingHeal.kt:195]: Done.

File: src/main/kotlin/org/wfanet/measurement/securecomputation/deploy/gcloud/spanner/InternalApiServer.kt (hunk 2 above)
Comment (Marco-Premier) [reply src/main/kotlin/org/wfanet/measurement/securecomputation/deploy/gcloud/spanner/InternalApiServer.kt:172]: Done.

**Review by Marco-Premier — COMMENTED** [review]
> @Marco-Premier made 4 comments.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4429)* status: 0 of 17 files reviewed, 4 unresolved discussions (waiting on stevenwarejones and stevenwarejones+AGNT).

File: docs/edpaggregator/deployment-guide.md (hunk 3 above)
Comment (stevenwarejones) [reply docs/edpaggregator/deployment-guide.md:661]: Not verified — the defaults and per-method wording are corrected, but the revised control-plane total still omits the dispatcher. `VidLabelingDispatcherFunction` constructs a `WorkItems` stub and passes it to `VidLabelingDispatchSequencer`, whose WorkItem creation calls run through `rpcThrottlers.controlPlane`. The envelope is therefore 28 processes × 4 QPS = 112 QPS, not 27 × 4 = 108 QPS. Update the calculation and remove the claim that the dispatcher does not call the control plane.

File: docs/edpaggregator/deployment-guide.md (hunk 4 above)
Comment (stevenwarejones) [reply docs/edpaggregator/deployment-guide.md]: Verified — the deployment guide now states that both VID Labeling scheduler jobs use a 660-second attempt deadline, preserving 60 seconds of headroom beyond the monitor function's 600-second timeout. This matches the Terraform configuration.

File: src/main/kotlin/org/wfanet/measurement/edpaggregator/tools/VidLabelingHeal.kt (hunk 1 above)
Comment (stevenwarejones) [reply src/main/kotlin/org/wfanet/measurement/edpaggregator/tools/VidLabelingHeal.kt:195]: Verified — `VidLabelingHeal` now applies `VidLabelingRpcDurationConverter` to all three recovery throttle flags. The reusable converter target is exposed to the tools package, and command-line tests cover both negative and partially malformed values.

File: src/main/kotlin/org/wfanet/measurement/securecomputation/deploy/gcloud/spanner/InternalApiServer.kt (hunk 2 above)
Comment (stevenwarejones) [reply src/main/kotlin/org/wfanet/measurement/securecomputation/deploy/gcloud/spanner/InternalApiServer.kt:172]: Verified — `InternalApiServer` now applies `VidLabelingRpcDurationConverter` to all three throttle flags. The reusable converter target is visible to the Secure Computation deployment package, and command-line tests cover both negative and partially malformed values.

**Review by stevenwarejones — CHANGES_REQUESTED** [review]
> @stevenwarejones+AGNT made 4 comments.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4429)* status: 0 of 17 files reviewed, 4 unresolved discussions (waiting on stevenwarejones).

**Review by stevenwarejones — COMMENTED** [review]
> @stevenwarejones reviewed 17 files and all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4429)* status: all files reviewed, 4 unresolved discussions (waiting on Marco-Premier).

**Review by stevenwarejones — COMMENTED** [review]
> @stevenwarejones resolved 3 discussions.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4429)* status: all files reviewed, 1 unresolved discussion (waiting on Marco-Premier).

**Review by stevenwarejones — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4429)* status: all files reviewed, 1 unresolved discussion (waiting on Marco-Premier).

File: docs/edpaggregator/deployment-guide.md (hunk 3 above)
Comment (Marco-Premier) [reply docs/edpaggregator/deployment-guide.md:661]: Done.

**Review by Marco-Premier — COMMENTED** [review]
> @Marco-Premier made 1 comment.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4429)* status: 16 of 17 files reviewed, 1 unresolved discussion (waiting on stevenwarejones and stevenwarejones+AGNT).

**Review by laureanobrs — APPROVED** [review]
> @laureanobrs reviewed 17 files and all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4429)* status: all files reviewed, 2 unresolved discussions (waiting on Marco-Premier, stevenwarejones, and stevenwarejones+AGNT).

**Review by Marco-Premier — COMMENTED** [review]
> @Marco-Premier reviewed 18 files and all commit messages, resolved 2 discussions, and dismissed @stevenwarejones and @stevenwarejones+AGNT from a discussion.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/4429)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on laureanobrs and stevenwarejones).
